### ISSUE TITLE
Cannot be installed with CraftCMS 4 beta

### ISSUE THREAD

**Issue by cornedor** [codemonauts/craft-redactor-nofollow#4]
```
Problem 1
    - Root composer.json requires codemonauts/craft-redactor-nofollow 1.0.1 -> satisfiable by codemonauts/craft-redactor-nofollow[1.0.1].
    - codemonauts/craft-redactor-nofollow 1.0.1 requires craftcms/cms ^3.0.0 -> found craftcms/cms[3.0.0-beta.1, ..., 3.7.38] but it conflicts with your root composer.json require (^4.0.0-beta.1).
```

**Comment by fleaz** [comment]
Hey @cornedor 
we are aware that our plugins are not yet compatible with Craft 4. But it's the next thing on our road map and we are already working on it :)

**Comment by kringkaste** [comment]
Hey @cornedor , we just released version 2.0.0 of the plugin with Craft 4 compatibility.

**Comment by cornedor** [comment]
Thanks 👌
